packages/quantic/force-app/examples/main/lwc/exampleInsightPanel/exampleInsightPanel.js-meta.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>56.0</apiVersion>
+    <isExposed>true</isExposed>
+    <targets>
+        <target>lightning__RecordPage</target>
+        <target>lightning__AppPage</target>
+        <target>lightning__HomePage</target>
+        <target>lightningCommunity__Page</target>
+        <target>lightningCommunity__Default</target>
+    </targets>
+    <targetConfigs>
+        <targetConfig targets="lightningCommunity__Default">
+            <property name="engineId" type="String" default="example-search" label="Enter Coveo Headless engine ID"/>
+            <property name="insightId" type="String" default="default" label="The insight panel config ID"/>
+        </targetConfig>
+        <targetConfig targets="lightning__AppPage, lightning__RecordPage, lightning__HomePage">
+            <property name="engineId" type="String" default="example-search" label="Enter Coveo Headless engine ID"/>
+            <property name="insightId" type="String" default="default" label="The insight panel config ID"/>
+        </targetConfig>
```

Comment:
Why are you using two identical target configs? Couldn't you use the same config for all targets?

```suggestion
        <targetConfig targets="lightning__AppPage, lightning__RecordPage, lightning__HomePage, lightningCommunity__Default">
            <property name="engineId" type="String" default="example-search" label="Enter Coveo Headless engine ID"/>
            <property name="insightId" type="String" default="default" label="The insight panel config ID"/>
        </targetConfig>
```

Reply:
It's been a while since I made the one for the exampleSearch but i'm about 90% sure it used to be that you needed two separate ones for Community builder and Console app builder. I just tried with your suggestion and it looks like it's working now. 👍 Nice!